#
# Plans a path, using the KOMO motion planner

# the name of the planning group.
# can be either 'left_arm' or 'right_arm'
string planning_group

# the name of the end effector link
string eef_link

# description of the target position
# 6 values mean [x,y,z,roll,pitch,yaw]
# 7 values mean [x,y,z,qx,qy,qz,qw]
# 3 values mean [x,y,z] (position only)
float64[] target
# if true, the trajectory will not be executed
bool plan_only
---
trajectory_msgs/JointTrajectory trajectory
bool result
